packages/vite/src/node/build.ts

```diff
@@ -329,6 +329,35 @@ export function resolveBuildPlugins(config: ResolvedConfig): {
   }
 }
 
+export function resolvePaths(config: ResolvedConfig, options: ResolvedConfig["build"]) {
+  const rollupOptions = options.rollupOptions
+  const libOptions = options.lib
+  const ssr = !!options.ssr
+
+  const resolve = (p: string) => path.resolve(config.root, p)
+
+  if (Array.isArray(rollupOptions?.input))
+    rollupOptions.input = rollupOptions.input.map(input => resolve(input))
```

Comment:
I think we should change `rollupOptions.input` while generating the ResolvedConfig in `resolveBuildOptions`. Or if we do it here, we need to avoid modifying the ResolvedConfig. WDYT about moving it to `resolveBuildOptions` ?